### ISSUE TITLE
How to finetune the segmoe and train lora

### ISSUE THREAD

**Issue by trouble-maker007** [segmind/segmoe#23]
Is there any finetune the segmoe and train lora tutorials in the future?

**Comment by Warlord-K** [comment]
Yes, we are working on finetuning the models, it might take a bit as we are working reducing the memory usage first.
